src/Widgets/AppContainers/InstalledPackageRowGrid.vala

```diff
@@ -56,6 +57,11 @@ public class AppCenter.Widgets.InstalledPackageRowGrid : AbstractPackageRowGrid
             return true;
         });
 
+        release_expander_label = new Gtk.Label ("");
```

Comment:
I don't know the history behind the changes in this file or the exact effect they have, but if they're intended to fix some issue that's been discovered, I'm sure the changes would be accepted upstream...